lib/Basics/StaticStrings.cpp

```diff
@@ -186,7 +186,10 @@ std::string const StaticStrings::AsyncId("x-arango-async-id");
 std::string const StaticStrings::Authorization("authorization");
 std::string const StaticStrings::BatchContentType(
     "application/x-arango-batchpart");
-std::string const StaticStrings::CacheControl("cache-control");
+std::string const StaticStrings::CacheControl("Cache-Control");
+std::string const StaticStrings::CacheControlValue(
+    "no-cache, no-store, must-revalidate, pre-check=0, post-check=0, "
+    "max-age=0, s-maxage=0");
```

Comment:
I would not define a static string for the value. It is only used once, and at the usage site it is not clear what the value actually is.

Reply:
Fixed in 9cae2f96bcc